inc/options/class-wpseo-option-wpseo.php

```diff
@@ -325,6 +326,7 @@ protected function validate_option( $dirty, $clean, $old ) {
 				case 'clean_permalinks_extra_variables':
 				case 'indexables_overview_state':
 				case 'dismiss_old_premium_version_notice':
+				case 'jetpack_ad_start_date':
```

Comment:
Really? Any value is fine here?
I'd think you'd want at the very least the string sanitize. And probably also the format check?

Reply:
Right: I added the validation: if it's not a valid date the default is taken.